pkg/maps/ctmap/ctmap.go

```diff
@@ -213,6 +213,10 @@ type Map struct {
 	// define maps to the macro used in the datapath portion for the map
 	// name, for example 'CT_MAP4'.
 	define string
+
+	// This field indicates which cluster this ctmap is. Zero for global
```

Comment:
nit: We should probably start revising our usage of "global" to mean "the local cluster", since global sounds confusingly like all clusters. I know the naming is grandfathered in since it was referring to "globally across the node" instead of "per-endpoint CT" so the naming is not super easy here, but there's lots of opportunity to further confuse readers by continuing to use the `global` phrasing going forward.